### PR TITLE
Import Flysystem patch to migrate fields to Flysystem

### PR DESCRIPTION
**Pull request by psoleckimoj**
- From here https://www.drupal.org/project/flysystem/issues/2691731
- https://groups.drupal.org/node/518975
- https://github.com/cweagans/composer-patches

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ RUN composer install \
 # Copy Project
 COPY modules/custom modules/custom
 COPY sites/ sites/
+COPY ./flysystem/FieldMigration.php /var/www/html/modules/contrib/flysystem/src/Form/FieldMigration.php
```
Comment (spikeheap) [inline Dockerfile]: <I don't have an emoji for "I'm being really nitpicky"> I think we can remove the preceding `./` for consistency with the lines above

File: flysystem/FieldMigration.php [hunk 2]
```diff
@@ -0,0 +1,414 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\flysystem\Form\FieldMigration.
+ */
```
Comment (spikeheap) [inline flysystem/FieldMigration.php:6]: Would it be worth us including the following?
- a link to the source of this patch
- license/credit to the author
- the purpose of the patch, and what it enables
- when it's safe to remove the patch (if ever)

Comment (psoleckimoj) [reply flysystem/FieldMigration.php:6]: @spikeheap still playing, not sure this is the right way to do it at the moment, you can apply patches using composer so messing with that atm :)

File: Dockerfile (hunk 1 above)
Comment (psoleckimoj) [reply Dockerfile]: Removed this line anyways :)

File: flysystem/FieldMigration.php (hunk 2 above)
Comment (psoleckimoj) [reply flysystem/FieldMigration.php]: Now patching from Drupal website

**Pull request merged**